.sti/bin/assemble

```diff
@@ -1,52 +1,63 @@
 #!/bin/bash
 
-# For SCL enablement
-source .bashrc
+# This script uses these environment variables:
+#
+# DISABLE_COLLECTSTATIC: if not empty, inhibits execution of 'manage.py collectstatic'.
 
-set -e
+function assemble() {
+  # For SCL enablement
+  source .bashrc
 
-echo "---> Copying application source ..."
-cp -Rf /tmp/src/* ./
+  set -e
 
-if [ -f setup.py ]; then
-    echo "---> Installing application ..."
-    python setup.py install --user
-elif [ -f requirements.txt ]; then
-    echo "---> Installing dependencies ..."
-    pip install --user -r requirements.txt
-fi
+  echo "---> Copying application source ..."
+  cp -Rf /tmp/src/* ./
```

Comment:
i know you didn't add this and we do it in all our scripts, but as a rule copying to WORKDIR really bothers me.  Seems like we ought to be explicit about where we're putting the app source.  @mfojtik @soltysh was there a reason we used this pattern that i'm forgetting?